requeue/requeue.go

```diff
@@ -58,6 +58,12 @@ func (r *RequeueSubsystem) requeueCommand(c *server.Connection, s *server.Server
 		_ = c.Error(cmd, err)
 		return
 	}
+	// Acknowledge will not return a job (nor an error) if there's no in-memory
```

Comment:
[nit] A little clearer this way. I would not have understood this comment if I hadn't read your PR description.

```suggestion
	// s.Manager().Acknowledge() will return (nil, nil) if there's no in-memory
```